official/resnet/keras/keras_imagenet_main.py

```diff
@@ -154,7 +153,7 @@ def run_imagenet_with_keras(flags_obj):
   # in the graph.
   tf.keras.backend.set_learning_phase(True)
 
-  model.compile(loss='categorical_crossentropy',
+  model.compile(loss=softmax_corss_entrophy,
```

Comment:
Let us also add a comment about the fact that we are using logits loss.

Reply:
I already changed the function name. Do you think the comment is still needed?